<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>GroupBox</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnGetfilename">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="le_filename">
            <property name="text">
             <string>P-129_out.LAS</string>
            </property>
           </widget>
          </item>
          <item row="15" column="2">
           <widget class="QPushButton" name="btn_plottops">
            <property name="text">
             <string>Plot Tops</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1" colspan="2">
           <widget class="QLineEdit" name="le_topname"/>
          </item>
          <item row="11" column="0" colspan="3">
           <widget class="QListWidget" name="tbl_tops"/>
          </item>
          <item row="14" column="2">
           <widget class="QPushButton" name="btn_addtop">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Top</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1" colspan="2">
           <widget class="QLineEdit" name="le_topdepth"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>File</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Plot1</string>
            </property>
           </widget>
          </item>
          <item row="10" column="2">
           <widget class="QPushButton" name="btnPlotlas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <widget class="QComboBox" name="cb_plot3"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Plot3</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QComboBox" name="cb_plot1"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Plot4</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="QComboBox" name="cb_plot2"/>
          </item>
          <item row="9" column="1" colspan="2">
           <widget class="QComboBox" name="cb_plot4"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Plot2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btn_loadlas">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="PlotWidget" name="gvPlot1"/>
       </item>
       <item>
        <widget class="PlotWidget" name="gvPlot2"/>
       </item>
       <item>
        <widget class="PlotWidget" name="gvPlot3"/>
       </item>
       <item>
        <widget class="PlotWidget" name="gvPlot4"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
